### PR TITLE
Changes following public Beta launch

### PR DESCRIPTION
**Pull request by Tomcowan1**
### Trello card

https://trello.com/c/zZB9Comq/1907-teaching-in-further-education-tife-page-changes

### Context

Less content is needed on this page, because FE has it's own site (just launched into private Beta). FE team want to add a line "You don't need a degree to teach in further education." No other new content.

### Guidance to review

Do not merge yet.

### CODE REVIEW

File: app/views/content/become-a-further-education-teacher.md [hunk 1]
```diff
@@ -23,57 +23,13 @@ Further education is any study that takes place after secondary school but outsi
...
-
-
-## Next steps
-
-Speak to an adviser on [0300 303 1877](tel:0300 303 1877) or by emailing feadviceline@etfoundation.co.uk.
-
-Search for [further education jobs](https://www.aocjobs.com/jobs/).
-
-Visit the [Teach in Further Education website](https://www.teach-in-further-education.campaign.gov.uk/).
+To find out more, visit the [teach in further education website]https://www.teach-in-further-education.campaign.gov.uk/
```
Comment (gemmadallmandfe) [inline app/views/content/become-a-further-education-teacher.md:33]:
```suggestion
To find out more, visit the [teach in further education website](https://www.teach-in-further-education.campaign.gov.uk/).
```

**Comment by gemmadallmandfe** [comment]
I think the reduced content makes sense @Tomcowan1 and means that all future content changes (eg bursaries) will only need to be made on the FE site.

One suggested change - link is not quite right. Other than that, happy to approve and then merge when appropriate.

**Comment by Tomcowan1** [comment]
Hi @gemmadallmandfe Huw's happy, as am I. So, when you're ready we're good to merge :-) .

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**